README.rst

```diff
@@ -80,19 +80,13 @@ https://molecule.readthedocs.io/
 Contact
 =======
 
-IRC
----
-
-Join us in the #molecule-users channel on `freenode`_.
+* Join us in the ``#molecule-users`` channel on `freenode`_.
+* Join the discussion in `molecule-users Forum`_
+* Want to know about releases, subscribe to `ansible-announce <https://groups.google.com/group/ansible-announce>`_
```

Comment:
Maybe move these links to the list below to keep the code style?

Reply:
Last time I asked we said we were keeping it as `#molecule-users`, though you are right to check. I've asked again for final confirmation.